Diary/Controller/DetailDiaryViewController.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  DetailDiaryViewController.swift
+//  Diary
+//
+//  Created by Kobe, Moon on 2023/09/02.
+//
+
+import UIKit
+
+class DetailDiaryViewController: UIViewController {
+    private let diaryTextView: UITextView = {
+        let textView = UITextView()
+        
+        return textView
+    }()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        view.backgroundColor = .systemBackground
+        navigationItem.title = DiaryDateFormatter.convertDate(Date(), Locale.current.identifier)
+        
+        addSubViews()
+        diaryTextViewConstraints()
+        
+        diaryTextView.delegate = self
+        diaryTextView.keyboardDismissMode = .onDrag
+    }
```

Comment:
추상화 레벨을 맞춘다면, 더 깔끔해보일 것 같습니다.
```suggestion
    override func viewDidLoad() {
        super.viewDidLoad()

        view.backgroundColor = .systemBackground
        configureNavigationItem() // 이런식으로 추상 레벨 통일
        
        addSubViews()
        diaryTextViewConstraints()
        
        diaryTextView.delegate = self
        diaryTextView.keyboardDismissMode = .onDrag
    }
    
    private func configureNavigationItem() {
         navigationItem.title = DiaryDateFormatter.convertDate(Date(), Locale.current.identifier)
    }
```

Reply:
### 🙋‍♂️ 세 번째 코멘트 답변
- 감사합니다. 조금 더 깔끔하고 좋은 코드를 만들기 위해 신경쓰겠습니다. 코드는 수정하도록 하겠습니다. 😀